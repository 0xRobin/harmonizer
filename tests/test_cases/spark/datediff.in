select datediff(day, '2023-01-01'::date, '2024-01-01'::date)