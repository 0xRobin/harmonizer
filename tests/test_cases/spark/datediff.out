/* Success! If you're still running into issues, check out https://dune.com/docs/query/syntax-differences/ or reach out in the #dune-sql Discord channel. */

SELECT
  DATE_DIFF(
    'day',
    CAST(SUBSTR(CAST(TRY_CAST('2023-01-01' AS DATE) AS VARCHAR), 1, 10) AS DATE),
    CAST(SUBSTR(CAST(TRY_CAST('2024-01-01' AS DATE) AS VARCHAR), 1, 10) AS DATE)
  )